.github/workflows/shisen_ci.yml

```diff
@@ -0,0 +1,22 @@
+name: Shisen CI
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+      
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    name : Build the Project
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
+    with:
+      node-version: '14'
```

Comment:
I think the error is in this line, the `with:` not aligned with the `uses:`.

Reply:
Will be fixed shortly, thank you for the suggestions